proxy/src/telemetry/metrics/latency.rs

```diff
@@ -5,33 +5,31 @@ use super::histogram::{Bounds, Bucket, Histogram};
 /// The maximum value (inclusive) for each latency bucket in
 /// tenths of a millisecond.
 pub const BOUNDS: &Bounds = &Bounds(&[
-    // The controller telemetry server creates 5 sets of 5 linear buckets
-    // each:
-    // prometheus.LinearBuckets(1, 1, 5),
+    Bucket::Le(1),
+    Bucket::Le(2),
+    Bucket::Le(3),
+    Bucket::Le(4),
+    Bucket::Le(5),
     Bucket::Le(10),
     Bucket::Le(20),
     Bucket::Le(30),
     Bucket::Le(40),
     Bucket::Le(50),
-    // prometheus.LinearBuckets(10, 10, 5),
     Bucket::Le(100),
     Bucket::Le(200),
     Bucket::Le(300),
     Bucket::Le(400),
     Bucket::Le(500),
-    // prometheus.LinearBuckets(100, 100, 5),
     Bucket::Le(1_000),
     Bucket::Le(2_000),
     Bucket::Le(3_000),
     Bucket::Le(4_000),
     Bucket::Le(5_000),
-    // prometheus.LinearBuckets(1000, 1000, 5),
     Bucket::Le(10_000),
     Bucket::Le(20_000),
     Bucket::Le(30_000),
     Bucket::Le(40_000),
     Bucket::Le(50_000),
-    // prometheus.LinearBuckets(10000, 10000, 5),
     Bucket::Le(100_000),
```

Comment:
TIOLI, but I don't think we necessarily need anything larger than 60 seconds in this list. As written the largest bucket is 8+ minutes. By comparison, the Prometheus go client's [default buckets](https://github.com/prometheus/client_golang/blob/master/prometheus/histogram.go#L54), which are intended to track network service response time, max out at 10 seconds.

Reply:
Yeah, I agree --- I think we ought to track that in a separate issue?

Reply:
If you change the subject of the PR, I think it's fine to include here. In fact, I think it's important that we remove buckets while we're adding them so we don't change our timeseries capacity significantly

Reply:
@olix0r Okay, will do --- do you agree that 60 seconds is a good upper bound for the histogram?